WordPress/src/main/res/layout/reader_recommended_blog_item.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/reader_interest_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="?attr/selectableItemBackground">
+
+    <androidx.appcompat.widget.AppCompatImageView
+        android:id="@+id/site_icon"
+        style="@style/ReaderImageView.Avatar"
+        android:layout_marginEnd="@dimen/reader_site_header_avatar_margin_end"
+        android:layout_marginTop="@dimen/margin_extra_large"
+        android:background="@drawable/bg_oval_placeholder"
+        android:contentDescription="@null"
+        android:importantForAccessibility="no"
+        android:src="@drawable/bg_oval_placeholder_globe_32dp"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="1" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_start"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_begin="@dimen/margin_extra_large" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_end"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_end="@dimen/margin_extra_large" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_top"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layout_constraintGuide_begin="@dimen/margin_extra_large" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_bottom"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layout_constraintGuide_end="@dimen/margin_extra_large" />
+
+    <org.wordpress.android.widgets.WPTextView
```

Comment:
ohh sure, I thought you want to use the custom text view. 

BTW. TextView and some other views are automatically replaced with Material components when using the Material theme, but this might now work with the Bridge theme 🤔

Reply:
Yeah, I'm not an expert but seems like they might not work with the Bridge theme. Thank you for replacing them with material components!